cpp/src/column/column.cu

```diff
@@ -177,7 +177,7 @@ struct create_column_from_view {
            std::enable_if_t<std::is_same<ColumnType, cudf::string_view>::value>* = nullptr>
  std::unique_ptr<column> operator()() {
    cudf::strings_column_view sview(view);
-   auto col = cudf::strings::detail::slice(sview, view.offset(), -1, 1, stream, mr);
+   auto col = cudf::strings::detail::slice(sview, 0, view.size(), 1, stream, mr); 
```

Comment:
At the end, this will gather with gather mask of sequence ranging from [0, size_of_the_view), if I am not wrong.

Reply:
gather is taking into account the offset https://github.com/rapidsai/cudf/blob/branch-0.12/cpp/src/strings/copying/copying.cu#L88, and that _seems_ to be the issue (where we are double counting the offset, without my change).